app/posts/apply-for-teacher-training/2020-04-27-improving-the-references-user-journey.md

```diff
@@ -0,0 +1,93 @@
+
+## Showing the right guidance at the right time
+
+At various points in the user journey we tell candidates:
+
+* [we’ll contact your referees after you submit your application](/apply-for-teacher-training/improving-the-references-user-journey#static-guidance-about-contacting-your-referees-when-you-submit)
+* [we'll send your application to your provider when your references are in](/apply-for-teacher-training/improving-the-references-user-journey#static-guidance-about-sending-the-application-to-the-provider-once-the-references-are-in)
+
+This is not the case if a candidate keeps their references when applying again.
+
+### Hypothesis
+
+If we show the right guidance at the right time
+Then candidates will understand what happens with their references
+
+### What this looks like
+
+Rather than having static sentences explaining the above, [we'll show the guidance only when you add or replace a referee](/apply-for-teacher-training/improving-the-references-user-journey#showing-guidance-at-the-relevant-point-when-replacing-referees).
+
+## Cutting out unnecessary steps
+
+There's an intermediate step between viewing your application menu and adding your referees. This takes the form of a webpage with guidance about how to choose your referees - but there's no evidence to say that it's needed.
+
+### Hypothesis
+
+If we remove the intermediate step, whether you're applying for the first time or applying again
+Candidates will still give suitable referees
+Because there’s similar guidance on the following page
+
+### What this looks like
+
+[Candidates go straight to add their references](http://localhost:8080/apply-for-teacher-training/improving-the-references-user-journey#candidates-go-straight-to-add-their-referee), rather than [having to go through an intermediate step](/apply-for-teacher-training/improving-the-references-user-journey#removing-the-intermediate-page-before-adding-referees).
+
+## Leading the candidate to the necessary actions
+
+If a candidate deletes one or both of their referees they have to replace them. However, the design doesn’t encourage you to add another referee.
+
+### Hypothesis
+
+If we make adding another referee the main call to action
+Candidates will know they need to do this
+
+### What this looks like
+
+[The button for adding another referee was not the clear call to action](/apply-for-teacher-training/improving-the-references-user-journey#the-button-for-adding-another-referee-was-not-the-clear-call-to-action).
+
+[It's now clearer that you need to add another referee when you delete one](/apply-for-teacher-training/improving-the-references-user-journey#making-it-clearer-that-you-need-to-add-another-referee), even if you don’t do it now.
+
+[We've also made it clear that you need 2 referees when you've deleted both of them](/apply-for-teacher-training/improving-the-references-user-journey#making-it-clearer-that-you-need-to-add-2-referees).
+
+## Marking the references section as complete when applying again
```

Comment:
Also it means it’ll be much faster for users to complete without having to go in and out of each task to mark it as complete.